### PR TITLE
Temporary solution for demo | chore(torchlib)

### LINKED ISSUE #493: TorchScript Graph: Support model > 2GB
Dependent on `torch.Graph._export_onnx`, large model export requires specifying a target path to store the model and weights proto. Right now this path is fixed as "".

https://github.com/microsoft/onnx-script/blob/4986a4be9f8ff1b0e2b7bdde738836574f90e900/onnxscript/function_libs/torch_aten/graph_building.py#L475

### PR DESCRIPTION
**Pull request by wschin**
Hack for #493 for demo. Will need a more proper solution.

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]

**Comment by justinchuby** [comment]
Merging for demo

File: onnxscript/function_libs/torch_lib/graph_building.py [hunk 1]
```diff
@@ -698,7 +699,11 @@ def to_model_proto(
             keep_initializers_as_inputs=False,
             custom_opsets={},
             add_node_names=True,
-            onnx_file_path="",
+            # TODO(wechi): Passing in this instead of reading from env.
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/graph_building.py:702]:
```suggestion
            # TODO(#493): Passing in this instead of reading from env.
```

File: onnxscript/function_libs/torch_lib/graph_building_test.py [hunk 2]
```diff
@@ -139,5 +141,44 @@ def test_add_initializer_allows_adding_the_same_tensor_twice_using_same_name(sel
...
+        # # of model parameters:
+        #  input_size x hidden_size + hidden_size +
+        #  hidden_size x output_size + output_size
+        # Thus, we have
+        #  (4 x 50000000 + 50000000 = 250000000) +
+        #  (50000000 x 10 + 10 = 500000010)
+        #  = 750000010 parameters
+        #  = 750000010 * 4 bytes
+        #  = 3 GB
+        batch_size, input_size, hidden_size, output_size = 1, 4, 50000000, 10
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_lib/graph_building_test.py:168]: Would this consume a lot of memory? We don't have much in the CI machines. Maybe skip this in CI too?

File: onnxscript/function_libs/torch_lib/graph_building.py (hunk 1 above)
Comment (wschin) [reply onnxscript/function_libs/torch_lib/graph_building.py:702]: Ok.

File: onnxscript/function_libs/torch_lib/graph_building_test.py (hunk 2 above)
Comment (wschin) [reply onnxscript/function_libs/torch_lib/graph_building_test.py:171]: I will change the size of this test to `small`. Although it won't test the actual functionality, it can still test the code to fetch env variable.

File: .github/workflows/main.yaml [hunk 3]
```diff
@@ -66,6 +66,12 @@ jobs:
           CATCH_ORT_SEGFAULT: "${{ matrix.os == 'ubuntu-latest' && '1' || '0' }}"
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v3
+      - name: Upload Test Results
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: dynamo-report (${{ matrix.name }}-${{ matrix.os }})
+          path: report_dynamo_export.sarif
```
Comment (BowenBao) [inline .github/workflows/main.yaml:74]: Is this intended change? looks unrelated to this pr

File: noxfile.py [hunk 4]
```diff
@@ -14,6 +14,7 @@
     "numpy==1.23.5",
     "typing_extensions",
     "beartype",
+    "tabulate",
```
Comment (BowenBao) [inline noxfile.py:17]: same here

File: .github/workflows/main.yaml (hunk 3 above)
Comment (wschin) [reply .github/workflows/main.yaml]: Reverted. Basically, dynamo_export call triggers several exceptions in CI but no call stack or details were provided. I will revert them and create another issue.

Comment (wschin) [reply .github/workflows/main.yaml]: Tracked by #934.

**Comment by wschin** [comment]
@justinchuby, I disable that test in CI because CI is not ready. That new line is always run by other to_model_proto calls, so it should be fine.
